packages/devtools_app/lib/src/debugger/debugger_screen.dart

```diff
@@ -9,6 +9,7 @@ import 'package:provider/provider.dart';
 import 'package:vm_service/vm_service.dart';
 
 import '../analytics/analytics.dart' as ga;
+import '../analytics/constants.dart' as analytics_constants;
```

Comment:
probably not following it elsewhere in devtools but I beleive this should be analyticsConstants or constants not analytics_constants although maybe I'm missing something in the style guide.

Reply:
underscores is per the style guide. See https://dart-lang.github.io/linter/lints/library_prefixes.html, which is triggered if you name as analyticsConstants